up42/workflow.py

```diff
@@ -229,7 +235,45 @@ def add_workflow_tasks(self, input_tasks: Union[List[str], List[Dict]]) -> None:
             f"{self.workflow_id}/tasks/"
         )
         self.auth._request(request_type="POST", url=url, data=input_tasks)
-        logger.info("Added tasks to workflow: %r", input_tasks)
+        logger.info(f"Added tasks to workflow: {input_tasks}")
+
+        if return_parameters:
+            default_parameters = self._get_default_parameters()
+            logger.info(f"Returning default workflow parameters: {default_parameters}")
+            return default_parameters
```

Comment:
Adding an optional return value for a function feels a bit weird - is it completely necessary?

Reply:
Agree, this is not the best way to do it. Was trying to provide a solution to provide the template parameters to the user directly, but the simplest solution for now is probably to provide that manually with `.get_templates()` etc. Will  remove this when revisiting the templates.